concent_api/core/message_handlers.py

```diff
@@ -954,6 +957,69 @@ def store_message(
     return stored_message
 
 
+def handle_send_subtask_results_verify(
+    request,
+    subtask_results_verify: message.concents.SubtaskResultsVerify
+):
+    subtask_results_rejected = subtask_results_verify.subtask_results_rejected
+    report_computed_task = subtask_results_rejected.report_computed_task
+    compute_task_def = report_computed_task.task_to_compute.compute_task_def
+    validate_id_value(
+        compute_task_def['subtask_id'],
+        'subtask_id'
+    )
+    validate_golem_message_task_to_compute(report_computed_task.task_to_compute)
+    validate_golem_message_subtask_results_rejected(subtask_results_rejected)
+
+    client_public_key = decode_client_public_key(request)
+    other_party_public_key = decode_other_party_public_key(request)
+    current_time = get_current_utc_timestamp()
+
+    if Subtask.objects.filter(
```

Comment:
State transitions are not being validated here. For example if a client sends this message when state is `FORCING REPORT` the handler will proceed and fail only on model validations. This will result in HTTP 500. There are validations in the model but they should be treated like a last line of defense or sanity check. Views need to check for this before we even get to these validations.

Reply:
Again, as discussed yesterday, there are no transition checks at all in the `views` (handling functions). There should be a separate story/issue for enhancing message handlers with state transtion checks.